app/controllers/journeys_controller.rb

```diff
@@ -7,4 +9,9 @@ def show
 
     @metadata.og.image = "journeys/#{@journey.image_name.presence || "placeholder.png" }"
   end
+
+  private def redirect_inactive_eu_application
```

Comment:
tento zapis `private xxx` som este nevidel. My to pouzivame normalne cez

```
private

def xxx
```

Reply:
Mozem zmenit.

My sme ale komplet presli na tento explicitny zapis, ak mas dlhu class, je to ovela prehladnejsie ako scrollovat hore dole a hladat private.